accepted/0000-registry-url-in-lock-file.md

```diff
@@ -0,0 +1,57 @@
+- Start Date: 2017-09-21
+- RFC PR:
+- Yarn Issue:
+
+# Summary
+
+The yarn.lock file includes information about the registry that it was generated against. It would
+be nice if there was no registry information associated with the lock file
+
+# Motivation
+
+We currently have two internal registries/mirrors. We generate the lock file against one we have in
+our development environment. When our code is deployed to our build/testing environment, we
+no longer have access to the original registry that was used to generate the lock file. This causes
+our download to error as yarn cannot retrieve the packages.
+
+The expected output is for yarn to recognise the registry URL supplied (via either cli flag or
+`.yarnrc` or `.npmrc`) and download from that if the registry used in the lock file is not available.
+
+
+# Detailed design
+
+Let me know if this isn't enough, first time writing an RFC.
+
+
+As a dev that generated their lock file against REGISTRY_A

```

Comment:
Here we need a more technical explanation with examples of yarn.lock and config parameter names and concrete URL examples.

Reply:
@bestander updated the PR. Let me know if more is needed